### PR TITLE
[one-cmds] Add new options to one-optimize

### PR DESCRIPTION
**Pull request by jinevening**
This adds three new options to one-optimize
--fuse_preactivation_batchnrom
--make_batchnorm_gamma_positive
--fuse_activation_function

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/4773

### CODE REVIEW

File: compiler/one-cmds/one-optimize [hunk 1]
```diff
@@ -56,6 +56,12 @@ def _get_parser():
         help='fuse BatchNorm op to Transposed Convolution op')
     circle2circle_group.add_argument(
         '--fuse_bcq', action='store_true', help='apply Binary Coded Quantization')
+    circle2circle_group.add_argument(
+        '--fuse_preactivation_batchnorm', action='store_true', help='fuse BatchNorm operators of pre-activations to Convolution operator')
```
Comment (seanshpark) [inline compiler/one-cmds/one-optimize:60]: can you please split the lines? like,
```
'--fuse_preactivation_batchnorm', 
action='store_true', 
help='fuse BatchNorm operators of pre-activations to Convolution operator')
```
if possible, let's put  help in 90 columns.

Comment (jinevening) [inline compiler/one-cmds/one-optimize:60]:
```suggestion
        '--fuse_preactivation_batchnorm',
        action='store_true',
        help='fuse BatchNorm operators of pre-activations to Convolution operator')
```

File: compiler/one-cmds/one-optimize [hunk 2]
```diff
@@ -56,6 +56,18 @@ def _get_parser():
...
+    circle2circle_group.add_argument(
+        '--fuse_preactivation_batchnorm',
+        action='store_true',
+        help='fuse BatchNorm operators of pre-activations to Convolution op')
+    circle2circle_group.add_argument(
+        '--make_batchnorm_gamma_positive',
+        action='store_true',
+        help="make negative gamma of BatchNorm to a small positive value (1e-10). \
+        Note that this pass can change the execution result of the model. \
+        So, use it only when the impact is known to be acceptable.")
```
Comment (mhs4670go) [inline compiler/one-cmds/one-optimize:68]: (optional) you can also use triple quote.

And, there is one thing you should know. `one-optimize` is a python script but it doesn't have `.py` extension, which prevent it from format checker recognizing it as a python script. So, you should manually check the format by `yapf`.
```
$ yapf -i compiler/one-cmds/one-optimize
```

Comment (jinevening) [reply compiler/one-cmds/one-optimize:68]: Thanks for the tip 👍

File: compiler/one-cmds/one-optimize (hunk 1 above)
Comment (jinevening) [reply compiler/one-cmds/one-optimize]: I've split the lines.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**